vocabulary-draft/introduction.md

```diff
@@ -0,0 +1,22 @@
+This document intends to describe a set of events related to different phases of a Continuous Delivery process. 
+These events are categorized by meaning and the phase where they are intenteded to be used. 
+These events are agnostic from any specific tool and are designed to fit a wide range of scenarios. 
+
+The phases covered by this proposal are:
+
+- **Source Code Version Control Events**: Events emitted by changes in source code or by the creation, modification or deletion of new repositories that hold source code.
+- **Continuous Integration Pipelines Events**: includes events related to building, testings, packaging and releasing software artifacts, usually binaries.
+- **Continuous Deployment Pipelines Events**: include events related with environments where the artifacts produced by the integration pipelines actually run. These are usually services running in a specific environment (dev, QA, production), or embedded software running in a specific hardware. 
+
+These phases can also be considered as different profiles of the vocabulary that can be adopted independently. 
+
+# Required Metadata for CD Events
+
+The following attributes are REQUIRED to be present in all the Events defined in this vocabulary:
+
+- **Event ID**: defines a unique identifier for the event
+- **Event Type**: defines a textual description of the event type, only event types described in this document are supported. All event types should be prefixed with `CD.`
```

Comment:
From old PR:
@erkist 
> The CloudEvents spec states "SHOULD be prefixed with a reverse-DNS name. The prefixed domain dictates the organization which defines the semantics of this event type.", which I guess would mean "foundation.cd.repository.created", but the term "foundation" is a bit verbose for every type, so I would agree with the choice of just "cd". Maybe we can ask whoever owns http://events.cd/ for their domain name (appears unused) so we could have "cd.events.repository.created" :)

@salaboy
> @erkist I am all in favour for that.. let's add to the discussion points next week. I thought about the prefix cdf as well as an alternative, another topic for discussion.

Reply:
I created a discussion for this: https://github.com/cdfoundation/sig-events/discussions/20

Reply:
I think for this PR we should remove the required prefix, and say that the format is being defined, perhaps link to the github discussion.

Reply:
@afrittoli can you please share the reason why?